setup-tests-pipeline.ps1

```diff
@@ -17,13 +17,13 @@ if(!$skipCliDownload)
 
   Write-Host "Downloading Functions Core Tools...."
   Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/3/latest/version.txt' -OutFile version.txt
-  Write-Host "Using Functions Core Tools version: $(Get-Content -Raw version.txt)"
+  Write-Host "Using Functions Core Tools version: pgopa-CLI"
   $version = "$(Get-Content -Raw version.txt)"
   Remove-Item version.txt
 
   if ($version -and $version.trim())
   {
-    $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/3/latest/Azure.Functions.Cli.win-x86.zip"
+    $env:CORE_TOOLS_URL = "https://pgopafunctestv2storage.blob.core.windows.net/cli/pgopa-cli.zip"
```

Comment:
Where can I find the storage account? Is this published by the Azure Functions Core Tools CI?

Reply:
We just pushed the latest host from Pragna branch as we did not release the latest host yet.
We will go back and update these values once we will release the core tools in few weeks.

Reply:
Yes, core tools ci builds are uploaded to 'functionsclibuilds'. I will revert this as soon as the latest host changes move to release